ansible_collections/arista/avd/roles/eos_designs/schemas/eos_designs.schema.yml

```diff
@@ -3859,7 +3859,6 @@ $defs:
                 convert_types:
                 - str
                 min: 1
```

Comment:
I suggest to remove the minimum value as well, since a user might use a negative number or zero, in case they wish to manipulate the algorithms.

Reply:
I didn't think of that... yes let's also remove the min!